config/cruisecontrol.properties

```diff
@@ -227,11 +233,17 @@ removal.history.retention.time.ms=1209600000
 # The maximum number of completed user tasks for which the response and access details will be cached.
 max.cached.completed.user.tasks=100
 
-# The maximum number of completed monitoring user tasks for which the response and access details will be cached.
-max.cached.completed.monitor.user.tasks=20
+# The maximum number of completed kafka monitoring user tasks for which the response and access details will be cached.
```

Comment:
With the current behavior change in the interpretation of configs, can we update the 
(1) comment / documentation for `max.cached.completed.user.tasks`
(2) update its value to a reasonable smaller value -- e.g. `100 / 4` (since we added 4 new smaller sets of values)?